src/components/AppNavigation/Settings/SettingsTimezoneSelect.vue

```diff
@@ -21,7 +21,7 @@
 
 <template>
 	<li class="settings-fieldset-interior-item settings-fieldset-interior-item--timezone">
-		<TimezoneSelect
+		<TimezonePicker
 			:additional-timezones="additionalTimezones"
 			:value="timezone"
 			@change="setTimezoneValue" />
```

Comment:
I don't think this will fire. You might need `@input="..."` instead

Reply:
i havent tested anything yet. I pushed only because i already start replacing the component